.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+In this pull request I've added ...
+
+<!-- Please mention all relevant module/task numbers. -->
+
+Taks 1
+virtualenv
```

Comment:
This template provides data to prefill your Pull Request's request. It should be kept as generic as possible. Please use the template as provided. 

However, it's a good thing to keep your pull requests informative, the information you've entered here could be added manually while opening a pull request from the next task onwards.

You can read more about PR templates here https://blog.axosoft.com/enhancing-pull-request-descriptions-templates/

Reply:
Thank you Filip.
I have reverted my changes to PR template

Reply:
Approved, looks good!